src/index.js

```diff
@@ -24,6 +24,128 @@ pdfjsLib.GlobalWorkerOptions.workerSrc = '/pdf.worker.min.js'
 // =========================================
 /* global HTMLElement, CustomEvent */
 
+window.customElements.define('masked-input-helper',
```

Comment:
damn this custom element is massive

Reply:
Yeah, there's a bunch of tiny edge cases :sweat_smile: maybe we should start considering moving these custom elements to separate files to declutter `index.js` a little bit